Makefile

```diff
@@ -1,111 +1,115 @@
-# make    					build the entire site (all languages)
-# make en						build the English edition (replace 'en' with 'fr' for French)
+# make	 					build the entire site (all languages)
+# make init					just install NPM dependencies
+# make en					build just the English edition (replace 'en' with 'fr' for French, etc.)
 # make clean				destroy built files
-# make reset				destroy built files and build the entire site
-# make watch_css		watches for stylus (CSS) edits and compiles it
+# make reset				destroy built files and build all languages from scratch
+# make watch_css			watches for stylus (CSS) edits and compiles it
 
-default: clean_tmp mkdirs copy_assets build_all finalize
+#----------------------------------------------------------------------
+# CONFIGURATION
+#----------------------------------------------------------------------
 
-#---------------------------------------------------------------------- 
-# ALIASES 
-#---------------------------------------------------------------------- 
+# Collect list of languages based on available locale files
+LANGUAGES = $(shell find ./source/locales/ -name '*.json' -execdir basename -s .json {} \;)
 
-# Binaries
-BIN = ./node_modules/.bin/
-LIVESCRIPT_BIN = $(BIN)lsc
-LIVESCRIPT_PARAMS = -cob
-STYLUS_BIN = $(BIN)stylus
-STYLUS_PARAMS = -c -u ./node_modules/nib/
-STYLUS_WATCH_PARAMS = -c -w source/stylesheets/screen.styl -u nib -o public/assets/css/
+# Collect list of static assets that get copied over
+ASSETS = $(shell ls source/assets)
```

Comment:
`ls` is primarily for human readable output and should be avoided in a scripting/programming context. I'd either use `find` or shell globbing/expansion. Here's also one example: https://www.gnu.org/software/make/manual/html_node/Shell-Function.html `files := $(shell echo *.c)`.

Also, debatable, but for variables set by shell functions, I'd usually assign them via `:=` (as the example above does) to avoid potential race conditions (although rather unlikely and not the case here afais).